### PR TITLE
Feat: slicing milestone aduan warga

### PR DESCRIPTION
**Pull request by rayfajars**
# FEAT
- add redirect to detail aduan for test view detail
- add slicing detail aduan
- add data constant status aduan
- add component text milestone dan card milestone
- add component icon svg 
- add icon for milestone aduan warga in folder static

# CHANGE
- refactor file name page detail

# PREVIEW
[screen-capture (1).webm](https://user-images.githubusercontent.com/58242304/223355683-145ac50e-870c-408b-8486-1ff539937205.webm)

# FIGMA
https://www.figma.com/file/gC4ZDcyCfTfdD2Y3Ygc1U9/Aduan-Warga?node-id=7203%3A156415&t=LgUa9fz2EwMRGf0L-0

# EVIDENCE

title: slicing milestone aduan warga
project: Jabar Super Apps
participants: @rayfajars @marsellavaleria19 @Ibwedagama @agunghide @naufalihsank @doohanas

### CODE REVIEW

**Review by marsellavaleria19 — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

File: components/Aduan/Milestone.vue [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            <span class="text-gray-800">oleh</span>
+            <b>{{ milestone.name }}</b>
+          </template>
+        </TextMilestone>
+
+        <template
+          v-if="
+            milestone.status === dataStatusMilestone.followUp.status ||
+              milestone.status === dataStatusMilestone.done.status
+          "
```
Comment (naufalihsank) [inline components/Aduan/Milestone.vue]: minor

maybe we can use some method here to make the condition more maintainable and readable

```js
methods: {
  isFollowUpOrDone(milestone, dataStatusMilestone) {
    return milestone.status === dataStatusMilestone.followUp.status
      || milestone.status === dataStatusMilestone.done.status
  }
}
```

File: components/Aduan/Milestone.vue [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            fillColor: '',
+            conditionAduan: ''
+          }
+      }
+    }
+  }
+}
+</script>
+
+<style></style>
```
Comment (naufalihsank) [inline components/Aduan/Milestone.vue]: minor

unused code here

**Review by naufalihsank — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: components/Aduan/Milestone.vue (hunk 1 above)
Comment (rayfajars) [reply components/Aduan/Milestone.vue]: i am change [here](https://github.com/jabardigitalservice/super-app-web/pull/42/commits/51159d5a995df1142202a50cb7d861b14a32534a)

File: components/Aduan/Milestone.vue (hunk 2 above)
Comment (rayfajars) [reply components/Aduan/Milestone.vue]: i am change [here](https://github.com/jabardigitalservice/super-app-web/pull/42/commits/51159d5a995df1142202a50cb7d861b14a32534a)

**Pull request merged**